eth-sent-to-tornado-cash/src/index.ts

```diff
@@ -0,0 +1,56 @@
+import { Finding, HandleTransaction, TransactionEvent } from "forta-agent";
+import { createFinding, isInArray } from "./agent.utils";
+import AccountCashRecord, { createCashIn } from "./account.cash.record";
+
+const TORNADO_ADDRESSES: string[] = [
+  "0x12d66f87a04a9e220743712ce6d9bb1b5616b8fc",
+  "0x47ce0c6ed5b0ce3d3a51fdb1c52dc66a7c3c2936",
+  "0x910cbd523d972eb0a6f4cae4618ad62622b39dbf",
+  "0xa160cdab225685da1d56aa342ad8841c3b53f291",
+];
+const VALUE_THRESHOLD: bigint = BigInt("100000000000000000000"); // 100 ETH
+const TIME_LIMIT: bigint = BigInt("86400"); // 1 day
+
+export const provideHandleTransaction = (
+  tornadoAddresses: string[],
+  valueThreshold: bigint,
+  timeLimit: bigint
```

Comment:
Sometimes you use the trailing comma and sometimes you don't, please add the missing ones or remove all of the. Also see line 54.

Reply:
Solved using prettier